benefits/core/templates/core/help.html

```diff
@@ -17,23 +17,24 @@ <h2 class="pt-8" id="what-is-cal-itp">{% translate "core.pages.help.about" %}</h
         <p class="pt-4">{% translate "core.pages.help.about.p[1]" %}</p>
 
         <h2 class="pt-8" id="payment-options">{% translate "core.pages.help.payment_options" %}</h2>
-        <p class="pt-4">{% translate "core.pages.help.payment_options.p[0]" %}</p>
-
+        <p class="pt-4">{% translate "eligibility.pages.start.modal.p[0]" %}</p>
+        <p class="pt-4">{% translate "eligibility.pages.start.modal.p[1]" %}</p>
         <p class="contactless-symbol pt-3">
           <img class="icon mx-auto d-block"
                width="40"
                height="50"
                src="{% static 'img/icon/contactless.svg' %}"
                alt="{% translate "core.icons.contactless" context "image alt text" %}" />
         </p>
-
-        <p class="pt-4">{% translate "core.pages.help.payment_options.p[1]" %}</p>
+        <p class="pt-4">{% translate "eligibility.pages.start.modal.p[2]" %}</p>
+        <p class="pt-4">{% translate "eligibility.pages.start.modal.p[3]" %}</p>
         <p class="pt-4">
-          {% blocktranslate with website="https://cash.app/help/us/en-us/14425-cal-transit" %}core.pages.help.payment_options.p[2][0]{{ website }}{% endblocktranslate %}
-          {% blocktranslate with website="https://venmo.com/about/debitcard/" %}core.pages.help.payment_options.p[2][1]{{ website }}{% endblocktranslate %}
+          {% if agency %}
```

Comment:
Note that, unlike the case for the Contactless Cards modal which is on the Eligibility Start page, the user might not have chosen an `agency` yet when they go to the Help page. There wouldn't be specific link that we could show.

Therefore, here we're only showing this sentence if the user has chosen an agency. 

An alternative would be for product/design to provide us the copy to use when we're not in the context of a specific agency.

Reply:
Should this `{% if %}` wrap the `<p>` tag then? Otherwise might this `<p>` be empty?

Reply:
Yes, good point! Amended the commit as ed1935dcc995bdc88e2448a054d8e444121032d2